subsys/net/ip/ipv6.c

```diff
@@ -262,7 +262,8 @@ static struct net_route_entry *add_route(struct net_if *iface,
 		return route;
```

Comment:
Typo in commit `net: route: Add support for route lifetime`, s/rotue/route/
Please check all the commits for this.

Reply:
Thanks, will fix.